src/main.rs

```diff
@@ -196,14 +197,21 @@ async fn main() -> Result<(), Box<dyn Error>> {
         });
     }
 
+    let global_db_dir = path::Path::new(&app_config.rocksdb_dir)
+        .join("global")
+        .to_string_lossy()
+        .into_owned();
+    let global_db = Arc::new(RocksDB::new(&global_db_dir));
+    global_db.open().unwrap();
```

Comment:
We should push all this into the store.

Reply:
Which store? I was planning to use the same db for both onchain events and fnames.

Reply:
The file currently called ingest_state